### PR TITLE
Add "What You Will Learn" section, specifying HTML, CSS and JS in public page

### PR DESCRIPTION
**Pull request by thombudel**
screen shot 2017-03-29 at 10 36 26

### CODE REVIEW

**Comment by thombudel** [comment]
- Add test
- Render on page

File: app/views/shared/_what_you_will_learn.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div class="jumbotron">
+  <h3 class="text-center">What You Will Learn</h3>
+  <p class="text-center">Professional developers technologies</p>
+
+  <div class="row text-center">
+    <div class="col-lg-4">
+      <img class="img-circle" src="https://www.w3.org/html/logo/downloads/HTML5_Logo_512.png" alt="HTML logo" width="140" height="140">
```
Comment (arnoFleming) [inline app/views/shared/_what_you_will_learn.html.erb]: usualy you don't deeplink to other server's resources. You never know what happens to them

File: spec/views/pages/attend.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "pages/attend.html.erb",  type: :view do
+
+    it "Should render what you will learn section" do
+      render "shared/what_you_will_learn.html.erb"
```
Comment (arnoFleming) [inline spec/views/pages/attend.html.erb_spec.rb]: dont render another page than the one you're describing, see comment in #54 or #45 about it

File: app/views/shared/_what_you_will_learn.html.erb (hunk 1 above)
Comment (arnoFleming) [inline app/views/shared/_what_you_will_learn.html.erb]: but for this week its fine

**Pull request merged**